lib/management/user.ts

```diff
@@ -83,6 +83,117 @@ const withUser = (sdk: CoreSdk, managementKey?: string) => ({
       ),
       (data) => data.users,
     ),
+  activate: (loginId: string): Promise<SdkResponse<UserResponse>> =>
+    transformResponse<SingleUserResponse, UserResponse>(
```

Comment:
What's the reason for specifying the types twice?
(on the function return type & on the transformResponse types)